### PR TITLE
Issue a warning about .dev TLDs

### PR DESCRIPTION
**Pull request by danhunsaker**
Google is cracking down. Warn users about `.dev` not working, and recommend `.local` for dev, and `.test` for dry-run, instead.

### CODE REVIEW

File: processors/app/dns/add.go [hunk 1]
```diff
@@ -39,6 +39,16 @@ func Add(envModel *models.Env, appModel *models.App, name string) error {
...
 	}
 
+	// issue a warning about `.dev` being unusable with Chrome
+	if name[len(name)-4:] == ".dev" {
+		tld := appModel.DisplayName()
+		if tld == "dry-run" {
+			tld = "test"
+		}
+
+		display.Warn("\nGoogle has been locking down use of the .dev TLD, and your app may not be accessible with this domain.\nTry using %s.%s instead\n", name[0:len(name)-4], tld)
```
Comment (glinton) [inline processors/app/dns/add.go]: I'd return that as an error (assuming the caller prints the error) so it is a hard restriction. Returning early would alleviate frustrations later when the warning is ignored and a user gets another hard error from chrome.

Comment (glinton) [reply processors/app/dns/add.go]: Thanks for adding this in, btw.

**Comment by danhunsaker** [comment]
Not necessarily a _good_ reason, but sometimes folks like to insist on doing things their way, so I opted for the route of least surprise. Will switch to a hard failure in just a moment. ☺

**Comment by jeroenvisser101** [comment]
hey folks, even though it seems like it's being added just fine, it's being thrown as an error:


```
Error   : Google has been locking down use of the .dev TLD, and your app may not be accessible with this domain.
```

which lead us to believe it wasn't added.

**Comment by danhunsaker** [comment]
It's _supposed to_ be an error, and not actually add the alias.

**Comment by jeroenvisser101** [comment]
@danhunsaker ah, my bad. I thought reading the PRs description that it would be a warning. Any way to disable this check? Only Chrome blocks using .dev (when not on https), right?

**Comment by danhunsaker** [comment]
Only Chrome blocks it for now, but the others are likely to follow suit rather quickly. Fact is that Google owns `.dev` now, so it really is their call what to do with it.

**Pull request merged**